BankManagment/UnitOfWork/Entity/BankAccount.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace BankManagmentDB.Entity
+{
+    public class BankAccount
+    {
+        public Guid Id { get; set; }
```

Comment:
Create an abstract base class, which contains Id property, and inherit all entities from it